source/tyr/tyr/resources.py

```diff
@@ -825,14 +834,22 @@ def map_args_to_instance(attr_name):
                 if equipment_provider:
                     instance.equipment_details_providers.append(equipment_provider)
                 else:
-                    return (
-                        {
-                            "message": "Couldn't set equipment providers - Provider '{}' isn't present in db".format(
-                                provider_id
-                            )
-                        },
-                        400,
+                    msg = "Couldn't set equipment providers - Provider '{}' isn't present in db".format(
+                        provider_id
                     )
+                    return {"message": msg}, 400
+
+            instance.ridesharing_services = []
+            for ridesharing_service_id in args.ridesharing_services:
+                ridesharing_service = models.RidesharingService.query.get(ridesharing_service_id)
+                if ridesharing_service:
+                    instance.ridesharing_services.append(ridesharing_service)
+                else:
+                    msg = "Couldn't set ridesharing services - '{}' isn't present in db".format(
+                        ridesharing_service_id
+                    )
+                    return {"message": msg}, 400
+
```

Comment:
Could you add some tests on how to associate a Ridesharing to one o more instances?

Reply:
ok